app/enterprise/1.3-x/brain-immunity/install-configure.md

```diff
@@ -69,6 +69,18 @@ The release-scripts given are:
 3. **with-redis.yml** - creates redis instance that collector app will use if redis instance not already provided.
 
 
+##### Collector App Environment Variables
+The Collector App relies on several environment variables that need to be configured well for proper function.  These are the most impactful environment variables that should be specified to match the overall Kong and deployment environment, and what impact the variable has on Collector App.
+
+These variables have defaults in the provided docker-compose.yml that assume deployment with docker-compose on the same network that Kong is deployed on.  However, if this is not the case for your planned deployment of Collector App, please adjust the variables in the docker-compose.yml or specify the value of the varaibles docker-compose up like ```ENVVAR=somevalue ENVVAR=anothervalue docker-compose up```
+
+1. **KONG_PROTOCOL**: The protocol that Kong Admin API can be reached at.  It's going to be either http or https.
+2. **KONG_HOST**: The host name that Kong Admin API can be reached at.  If deploying with Docker Compose, this is going to be the name of the Kong container specified in the compose file.  If Kong Admin has been exposed behind a web address, KONG_HOST will be that web address.
+3. **KONG_PORT**: The port where Kong Admin can be found.  Together with KONG_PROTOCOL and KONG_HOST, Collector App is able to communicate with Kong Admin
```

Comment:
This sentence sounds like it cuts off short - I'm not sure if there's supposed to be something else here? If not, then maybe rephrase to:  

Collector App requires this setting, along with KONG_PROTOCOL and KONG_HOST, to communicate with Kong Admin.